docs/source/gbd2017_models/causes/ischemic_heart_disease/index.rst

```diff
@@ -132,11 +132,14 @@ State and Transition Data Tables
    * - State
      - State Name
      - Definition
+   * - S
+     - **S**\ usceptible
+     - Susceptible to IHD
    * - A
-     - Acute Myocardial Infarction (MI)
-     - Simulant that experiences acute MI symptoms listed in 'Myocardial infarction (MI) in GBD 2017'
-   * - C
-     - Chronic IHD
+     - **A**\ cute Myocardial Infarction (MI)
+     - Simulant that experiences acute MI symptoms 
+   * - P
+     - **P**\ ost-MI IHD
```

Comment:
Is this encapsulating everyone who has had MI, and is not currently experiencing MI, and they are only presumed to have those symptoms for the first 28 days? Or do they just stay in this box for 28 days, and then go back to susceptible?

Reply:
There is some residual health impact after the 28 days, so we don't put them back into S after an MI - no matter how long it has been.